diktat-analysis.yml

```diff
@@ -0,0 +1,114 @@
+- name: DIKTAT_COMMON
```

Comment:
why do we have so many overrides for rules in our own project?
I understand the configuration for domain name and srcDirectory, but we override the rest?

Reply:
Removed rule configuration overrides that actually do not override anything